README.md

```diff
@@ -1,3 +1,4 @@
-# Horse-club
-Design in Figma
-https://www.figma.com/file/3XHXfM8QHj5xseLsIrzjxs/%D0%9A%D0%BE%D0%BD%D0%BD%D1%8B%D0%B9-%D0%BA%D0%BB%D1%83%D0%B1-(Copy)?node-id=0%3A1
+
+<h1 align="center"><a href="https://ilyadevn.github.io/Horse-club/" target="_blank">Horse-club</a></h1>
```

Comment:
wtf dude, I told you to read what is markdown syntax, don't use html for it

Reply:
I read that I can use here markdown and html. Both ways are correct. I read it in different articles.

Reply:
yes, but use markdown, you need about 20 minutes to study it, you will use it in future

Reply:
Done.

Reply:
Done.